features/identity-core/org.wso2.carbon.identity.core.server.feature/pom.xml

```diff
@@ -110,6 +118,8 @@
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.base</bundleDef>
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.core</bundleDef>
                                 <bundleDef>org.wso2.carbon.identity.framework:org.wso2.carbon.identity.application.common</bundleDef>
+                                <bundleDef>ua.parser.wso2:ua-parser:${ua_parser.version}</bundleDef>
+                                <bundleDef>commons-collections.wso2:commons-collections:${apache.common.collection.version}</bundleDef>
```

Comment:
No need to pack commons-collections.wso2:commons-collections, as it is already packed in the product. Pls check

Reply:
Resolved